Routinator 0.8.0 fails to build without default features

Hi,

rustc fails to build routinator 0.8.0 with this error:
```
     Running `rustc --crate-name routinator --edition=2018 src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=3 -C panic=abort -Cembed-bitcode=no -C metadata=9b57c53ac75970a5 -C extra-filename=-9b57c53ac75970a5 --out-dir /var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps -L dependency=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps --extern bytes=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libbytes-9f76dfbb0debec0d.rmeta --extern chrono=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libchrono-540c88a9b3f39080.rmeta --extern clap=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libclap-d6009f0bae961b1a.rmeta --extern crossbeam_queue=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libcrossbeam_queue-5e40ac76a6cb8bc0.rmeta --extern crossbeam_utils=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libcrossbeam_utils-12c72ed85314f23c.rmeta --extern dirs=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libdirs-903c2cc4193bb44e.rmeta --extern fern=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libfern-d043b6fecd5d102d.rmeta --extern futures=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libfutures-4484e1f07c617124.rmeta --extern hyper=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libhyper-f492a8ae4d749b6a.rmeta --extern listenfd=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/liblistenfd-7ee7bb1d15df2d0b.rmeta --extern log=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/liblog-43b3bf5eea943ae6.rmeta --extern log_reroute=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/liblog_reroute-9ca6f5888a47362d.rmeta --extern nix=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libnix-75ed1c118adc10ef.rmeta --extern num_cpus=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libnum_cpus-f1d66017fb1ef0d6.rmeta --extern rand=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/librand-6b6c772dc892aa56.rmeta --extern reqwest=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libreqwest-a7ad09a326897179.rmeta --extern ring=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libring-762c58a632629655.rmeta --extern rpki=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/librpki-6ba43d92579fbcad.rmeta --extern rpki_rtr=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/librpki_rtr-8293b21bc33778da.rmeta --extern serde=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libserde-5d8c719ba953c43f.rmeta --extern serde_json=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libserde_json-00aa906138630bce.rmeta --extern slab=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libslab-cfd51fd3df660f95.rmeta --extern syslog=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libsyslog-380134f843bab924.rmeta --extern tempfile=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libtempfile-40079ce871f9f550.rmeta --extern tokio=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libtokio-3374595d4dcb49e6.rmeta --extern toml=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libtoml-f3f06665976386d7.rmeta --extern uuid=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libuuid-39cc7ea0310f0e47.rmeta -L native=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/build/ring-a7e542ec9686a34c/out`
error[E0432]: unresolved import `reqwest::Certificate`
  --> src/rrdp/http.rs:10:15
   |
10 | use reqwest::{Certificate, Proxy, StatusCode};
   |               ^^^^^^^^^^^ no `Certificate` in the root

error[E0599]: no method named `add_root_certificate` found for struct `reqwest::blocking::ClientBuilder` in the current scope
  --> src/rrdp/http.rs:74:31
   |
74 |             builder = builder.add_root_certificate(
   |                               ^^^^^^^^^^^^^^^^^^^^ method not found in `reqwest::blocking::ClientBuilder`

error: aborting due to 2 previous errors

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `routinator`.

Caused by:
  process didn't exit successfully: `rustc --crate-name routinator --edition=2018 src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=3 -C panic=abort -Cembed-bitcode=no -C metadata=9b57c53ac75970a5 -C extra-filename=-9b57c53ac75970a5 --out-dir /var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps -L dependency=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps --extern bytes=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libbytes-9f76dfbb0debec0d.rmeta --extern chrono=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libchrono-540c88a9b3f39080.rmeta --extern clap=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libclap-d6009f0bae961b1a.rmeta --extern crossbeam_queue=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libcrossbeam_queue-5e40ac76a6cb8bc0.rmeta --extern crossbeam_utils=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libcrossbeam_utils-12c72ed85314f23c.rmeta --extern dirs=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libdirs-903c2cc4193bb44e.rmeta --extern fern=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libfern-d043b6fecd5d102d.rmeta --extern futures=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libfutures-4484e1f07c617124.rmeta --extern hyper=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libhyper-f492a8ae4d749b6a.rmeta --extern listenfd=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/liblistenfd-7ee7bb1d15df2d0b.rmeta --extern log=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/liblog-43b3bf5eea943ae6.rmeta --extern log_reroute=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/liblog_reroute-9ca6f5888a47362d.rmeta --extern nix=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libnix-75ed1c118adc10ef.rmeta --extern num_cpus=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libnum_cpus-f1d66017fb1ef0d6.rmeta --extern rand=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/librand-6b6c772dc892aa56.rmeta --extern reqwest=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libreqwest-a7ad09a326897179.rmeta --extern ring=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libring-762c58a632629655.rmeta --extern rpki=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/librpki-6ba43d92579fbcad.rmeta --extern rpki_rtr=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/librpki_rtr-8293b21bc33778da.rmeta --extern serde=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libserde-5d8c719ba953c43f.rmeta --extern serde_json=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libserde_json-00aa906138630bce.rmeta --extern slab=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libslab-cfd51fd3df660f95.rmeta --extern syslog=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libsyslog-380134f843bab924.rmeta --extern tempfile=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libtempfile-40079ce871f9f550.rmeta --extern tokio=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libtokio-3374595d4dcb49e6.rmeta --extern toml=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libtoml-f3f06665976386d7.rmeta --extern uuid=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/deps/libuuid-39cc7ea0310f0e47.rmeta -L native=/var/tmp/portage/net-misc/routinator-0.8.0/work/routinator-0.8.0/target/release/build/ring-a7e542ec9686a34c/out` (exit code: 1)
```

And here is the full build output: https://paste.swordarmor.fr/raw/fwTz

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,7 +27,7 @@ log             = "0.4.8"
 log-reroute     = "0.1.5"
 num_cpus        = "1.12.0"
 rand            = "0.8.1"
-reqwest         = { version = "0.11.0", default-features = false, features = ["blocking", "gzip"] }
+reqwest         = { version = "0.11.0", default-features = false, features = ["blocking", "gzip", "default-tls" ] }
 ring            = "0.16.12"
 rpki            = { git = "https://github.com/NLnetLabs/rpki-rs.git", features = [ "repository", "rrdp", "rtr" ] }
 serde           = { version = "1.0.95", features = [ "derive" ] }
@@ -49,7 +49,7 @@ syslog          = "5.0.0"
 rustc_version   = "0.2.3"
 
 [features]
-default = ["rustls-tls", "socks", "ui"]
+default = [ "rustls-tls", "socks", "ui"]
 extra-debug = ["rpki/extra-debug"]
 socks = [ "reqwest/socks" ]
 rta = []

